src/components/feedbackFaces.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import { event } from "@site/src/utils/gtags.client";
+
+// Default export function component
+export default function FeedbackFaces() {
```

Comment:
Best component name ever haha

Reply:
I used to make games.  I still do, but I used to too. I don't think I'll ever top getting to implement a function called

`ExecuteWarpJump`